### DISCUSSION TITLE
作者大大，请问为什么运行扫码登录报错

### DISCUSSION THREAD

**Discussion by purechow** [RayWangQvQ/BiliBiliToolPro#368] [General]
repo目录: /ql/data/repo
bili仓库目录: /ql/data/repo/raywangqvq_bilibilitoolpro

bili_task_login.sh: line 9: dotnet: command not found

**Comment by RayWangQvQ** [comment]
因为爱情

**Reply by purechow** [reply]
![image](https://user-images.githubusercontent.com/106293702/208249596-6397d694-2262-4746-a16b-f8fd102cf06b.jpeg)

**Comment by qingsky7** [comment]
同问

**Comment by grinxxoo** [comment]
同错误 应该是dotnet的问题 但不会修

**Reply by fomodj** [reply]
同问

**Comment by Akunxin** [comment]
安装dotnet啊

**Comment by RayWangQvQ** [comment]
之前是通过编辑青龙的extra.sh来安装dotnet的，所有提示安装的，要不就是没改extra.sh，要不是就是改了没重启或手动触发。

最近版本已经改了，会在每个脚本里检测并安装，拉库更新下，再运行就行了。
